ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+
+"""
+ndmg.stats.qa_skullstrip
+~~~~~~~~~~~~~~~~~~~~~~
+The top level ndmg quality assurance for skull strip module.
+In this module, ndmg:
+1. Input original t1w file and the skull-striped brain file
```

Comment:
`skull-stripped`, not `skull-striped`. Two p's.

Reply:
fixed it